server.js

```diff
@@ -46,6 +80,16 @@ app.get('*', (request, response) => {
   response.status(404).send('these are not the droids you are looking for...');
 });
 
-
 // tell our server to start listening for requests
-app.listen(PORT, () => console.log(`listening on port ${PORT}`));
\ No newline at end of file
+app.listen(PORT, () => console.log(`listening on port ${PORT}`));
```

Comment:
I believe the app.listen needs to be at the bottom of the file?
It may cause issues with the class.

Reply:
Good feedback here too @Shane-Patrick-Roach 💯 

- Reformatting this is a great idea primarily to follow best practices 👍  Shane touched on a cool thing here though because the order of operations when dealing with wrapper functions and async code matters. Sometimes what can happen our HTTP requests can be received before the app's routes are even set up/configured. This can cause some unwanted behavior :)
- I believe classes are ideally top level (I could be wrong here though and we don't have to worry about that for now).